pinot-core/src/test/java/org/apache/pinot/queries/ForwardIndexHandlerReloadQueriesTest.java

```diff
@@ -480,6 +490,47 @@ public void testSelectWithAggregationQueriesWithReload()
         }));
     List<Object[]> beforeResultRows3 = resultTable.getRows();
 
+    // TEST4
+    query = "SELECT MAX(ARRAYLENGTH(column6)) from testTable LIMIT 10";
+    brokerResponseNative = getBrokerResponse(query);
+    assertTrue(brokerResponseNative.getProcessingExceptions() == null
+        || brokerResponseNative.getProcessingExceptions().size() == 0);
+    resultTable = brokerResponseNative.getResultTable();
+    assertEquals(brokerResponseNative.getNumRowsResultSet(), 1);
+    assertEquals(brokerResponseNative.getTotalDocs(), 400_000L);
+    assertEquals(brokerResponseNative.getNumDocsScanned(), 400_000L);
+    assertEquals(brokerResponseNative.getNumSegmentsProcessed(), 4L);
+    assertEquals(brokerResponseNative.getNumSegmentsMatched(), 4L);
+    assertEquals(brokerResponseNative.getNumEntriesScannedPostFilter(), 400000);
+    assertEquals(brokerResponseNative.getNumEntriesScannedInFilter(), 0L);
+    assertNotNull(brokerResponseNative.getProcessingExceptions());
+    assertEquals(brokerResponseNative.getProcessingExceptions().size(), 0);
+    assertEquals(resultTable.getDataSchema(), new DataSchema(new String[]{"max(arraylength(column6))"},
+        new DataSchema.ColumnDataType[]{DataSchema.ColumnDataType.DOUBLE}));
+    List<Object[]> beforeResultRows4 = resultTable.getRows();
+
+    // TEST5
+    query = "SELECT column1, max(column1), sum(column10) from testTable WHERE column6 = 1001 GROUP BY "
```

Comment:
Is column6 the column where dictionary is getting deleted ?

May be compare results for the same query before with dict and after without dict ? Also might want to double check in debugger once that query plan uses scan based evaluator and inv index is no longer being used (dropped infact)

Reply:
Yes. We do compare that the  query results exactly match before and after reload. 

The list of operations is documented in L726

>    * column1 -> change compression.
>    * column6 -> disable dictionary
>    * column9 -> disable dictionary
>    * column3 -> Enable dictionary.
>    * column2 -> Enable dictionary. Add inverted index.
>    * column7 -> Enable dictionary. Add inverted index.
>    * column10 -> Enable dictionary.